spine/model/layer/cnn/ppn.py

```diff
@@ -505,6 +579,8 @@ def forward(self, ppn_label, ppn_points, ppn_masks, ppn_layers, ppn_coords,
             Set of coordinates at the very last layer of the PPN
         ppn_classify_endpoins : TensorBatch, optional
             Set of logits associated with end point classification
+        frag_label : TensorBatch
+            (N, 1 + D + 1) Tensor of fragment labels for the batch
```

Comment:
Match doctring to other `clust_label` docstring

Reply:
Done.